src/ros_filter.cpp

```diff
@@ -634,22 +644,21 @@ void RosFilter::loadParams()
   twist_var_counts[StateMemberVyaw] = 0;
 
   // Determine if we'll be printing diagnostic information
-  node_->get_parameter_or("print_diagnostics", print_diagnostics_, false);
+  print_diagnostics_ = node_->declare_parameter("print_diagnostics", false);
 
   // Check for custom gravitational acceleration value
-  node_->get_parameter("gravitational_acceleration",
-    gravitational_acceleration_);
+  gravitational_acceleration_ = node_->declare_parameter("gravitational_acceleration",
```

Comment:
This can change back to `gravitational_acceleration_` since that's set to 9.8... in the constructor